test/tests/functional/pbs_job_array.py

```diff
@@ -39,12 +39,50 @@
 
 
 from tests.functional import *
+from ptl.utils.pbs_logutils import PBSLogUtils
 
 
 class TestJobArray(TestFunctional):
     """
     Test suite for job array feature
     """
+    lu = PBSLogUtils()
+    qjh = """
+import pbs
+
+e = pbs.event()
+j = e.job
+if j.max_run_subjobs is None:
+    j.max_run_subjobs = %d
```

Comment:
This just made think of something.  This will set max_run_subjobs on all jobs, not just subjobs.  Maybe there should be an action function which rejects it for non-job arrays?

Reply:
I'll add action functions